src/rust/rust-proto/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+pub(crate) mod graplinc {
+    pub(crate) mod grapl {
+        pub(crate) mod api {
+            pub(crate) mod graph {
+                pub(crate) mod v1beta1 {
+                    include!(concat!(
+                        env!("OUT_DIR"),
+                        "/graplinc.grapl.api.graph.v1beta1.rs"
```

Comment:
This obviously isn't where this term is defined, but since the corresponding `.proto` file wasn't changed in this PR I'll comment here. I'm having a little trouble understanding what `api` means in this package name. These data don't seem to be RPC APIs, instead they seem like pipeline data. It would be a good time to flesh out a package structure that makes sense.